samples/secure-fraud-detection/src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+quarkus.langchain4j.openai.timeout=60s
+quarkus.langchain4j.openai.chat-model.temperature=0
+quarkus.langchain4j.openai.api-key=${OPENAI_API_KEY}
+
+quarkus.oidc.provider=google
+quarkus.oidc.client-id=${GOOGLE_CLIENT_ID}
+quarkus.oidc.credentials.secret=${GOOGLE_CLIENT_SECRET}
+quarkus.oidc.authentication.redirect-path=/login
+
+#quarkus.langchain4j.openai.log-requests=true
+#quarkus.langchain4j.openai.log-responses=true
```

Comment:
I'd suggest uncommenting these two properties, they are generally quite useful for experimenting